webdriver/tests/perform_actions/wheel.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+from webdriver.error import NoSuchWindowException
+
+from tests.perform_actions.support.refine import filter_dict, get_events
+from tests.support.asserts import assert_move_to_coordinates
+
+
+def test_null_response_value(session, wheel_chain):
+    value = wheel_chain.scroll(0, 0, 0, 10).perform()
+    assert value is None
+
+
+def test_no_browsing_context(session, closed_window, wheel_chain):
```

Comment:
Please add another test for the top-level browsing context similar to what I have done here: https://hg.mozilla.org/integration/autoland/rev/12fadd0009e4. These changes will get a new PR today and getting merged soon by @jgraham. I simply don't want that we miss to add the appropriate check.

Reply:
Do you mean "test_no_top_browsing_context"? I added this test.